spk/owncloud/src/bin/owncloud-daemon

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+# ownCloud service setup
+WEB_DIR="/var/services/web_packages"
+# for backwards compatability
+if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ];then
+    WEB_DIR="/var/services/web"
+    if [ -z ${SYNOPKG_PKGNAME} ]; then
+        SYNOPKG_PKGNAME="owncloud"
+    fi
+fi
+PHP="/usr/local/bin/php74"
+OWNCLOUD="${PHP} ${WEB_DIR}/${SYNOPKG_PKGNAME}/occ background:cron"
+HOME_DIR="${SYNOPKG_PKGVAR}"
+# for backwards compatability
+if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ];then
+    HOME_DIR="${SYNOPKG_PKGDEST}/var"
+fi
+SLEEP_TIME="1800"
+
+# Main loop
+while true; do
+    # Update
+    echo "Updating..."
+    ${OWNCLOUD} >> "${HOME_DIR}/daemon.log" 2>&1
+
+    # Wait
+    echo "Waiting ${SLEEP_TIME} seconds..."
+    sleep ${SLEEP_TIME}
+done
```

Comment:
Just a thought, there is a reference to `crond.service` in the user manual... wonder if that couldn't be replaced by using it?  Although nowhere I see how to actually make usage of it... 

However I did found something interesting here (duno if it is worth investing time?) https://www.reddit.com/r/synology/comments/10iokvu/description_of_synocrond_tasks/?onetap_auto=true

Reply:
Hmm, I'm not sure trying to setup anything in `crond` is worthwhile since no real action is required for the website to be visible. I was looking at the cops package that @smaarn mentioned and there doesn't seem to be any daemon at all. Maybe if I just remove all of it?

Reply:
Actually the package to look at would be `tt-rss` if looking for a daemon service to run.

Reply:
Similar to my other comment on `SERVICE_COMMAND`, I looked at `tt-rss`, and the daemon file there includes a `--daemon` option as part of the execution. Unfortunately, the OCC utility does not have such an option which is why I suspect the original author used this while loop to keep the execution running. Can't think of another way to get a PID file so I'll leave it as is for now.

Reply:
This has been removed in https://github.com/SynoCommunity/spksrc/pull/5912 and is more aligned to the webservice example shown in https://github.com/SynoCommunity/spksrc/tree/master/spk/demowebservice.